bench/linux/optimized/i8042.ll

```diff
@@ -1641,9 +1641,9 @@ define internal fastcc noundef i32 @i8042_flush() unnamed_addr #0 align 16 {
 ; Function Attrs: fn_ret_thunk_extern nounwind null_pointer_is_valid
 define internal fastcc noundef i32 @i8042_create_aux_port(i32 noundef %0) unnamed_addr #0 align 16 {
   %2 = icmp slt i32 %0, 0
-  %3 = add nsw i32 %0, 2
+  %3 = add nuw nsw i32 %0, 2
   %4 = select i1 %2, i32 1, i32 %3
-  %5 = zext nneg i32 %4 to i64
+  %5 = sext i32 %4 to i64
```

Comment:
There are a couple of regressions in sext -> zext nneg folds like this.

Reply:
Issue occurs in CVP itself: https://llvm.godbolt.org/z/raz9Ee9rE

I guess it's kind of unavoidable: With nuw nsw there are two holes in the range, and we can only represent one of them.